grunt.js

```diff
@@ -348,25 +353,17 @@ module.exports = function(grunt) {
     'zip:dist'
   ]);
 
-  // Remove this and add to the test task once mmcc's coverall changes are merged
-  grunt.registerTask('newtest', ['build', 'karma:chrome']);
-
-  // Default task.
-  grunt.registerTask('default', ['build', 'test-local']);
+  // Default task - build and test
+  grunt.registerTask('default', ['test']);
 
-  // Development watch task. Doing the minimum required.
-  grunt.registerTask('dev', ['jshint', 'sass', 'browserify:build', 'karma:chrome']);
+  grunt.registerTask('test', ['build', 'karma:defaults']);
 
-  // Skin development watch task.
-  grunt.registerTask('skin-dev', ['connect:dev', 'watch:skin']);
+  // Run while developing
+  grunt.registerTask('dev', ['build', 'connect:dev', 'concurrent:watchSandbox']);
```

Comment:
@gkatsev that's related to my previous note
> I should also note, I tried combining ALL the watch tasks into one concurrent command, but it was a mess of log output. I think it probably makes more sense just to open a new terminal for other tasks. I welcome any ideas for improving from here.

I'll add it in and you can try it and let me know what you think.